src/erlcloud_sns.erl

```diff
@@ -286,8 +288,8 @@ list_endpoints_by_platform_application(PlatformApplicationArn, NextToken, Access
     list_endpoints_by_platform_application(PlatformApplicationArn, NextToken, new_config(AccessKeyID, SecretAccessKey)).
 
 -spec list_topics/0 :: () -> [{topics, [[{arn, string()}]]} | {next_token, string()}].
```

Comment:
Line 290 is too long: -spec list_topics/0 :: () -> [{topics, [[{arn, string()}]]} | {next_token, string()}]..